helm-chart/binderhub/templates/configmap.yaml

```diff
@@ -19,6 +19,19 @@ data:
   binder.repo2docker-image: {{ .Values.repo2dockerImage | quote }}
   binder.hub-url: {{ .Values.hub.url | quote }}
   binder.base_url: {{ .Values.baseUrl | quote }}
+  {{ if .Value.use_as_federation_portal -}}
+  binder.use_as_federation_portal: {{ .Value.use_as_federation_portal }}
+  {{- end }}
+  binder.cannonical_address: {{.Values.cannonical_address | quote }}
+  {{ if .Value.list_cookie_set_binders -}}
+  binder.list_cookie_set_binders: {{ .Values.list_cookie_set_binders }}
+  {{- end }}
+  {{ if .Value.default_binders_list-}}
+  binder.default_binders_list:
+  {{ range $value := .Values.default_binders_list -}}
+{{'-'+$value | indent 4 }}
```

Comment:
This should be:

{{ toYaml .Values.defaultBindersList | indent 4 }}

Reply:
You also need to explicitly load these in binderhub_config.py file with get_config.